jenkins/bootstrap_test.py

```diff
@@ -1975,9 +1976,20 @@ def testNoBadVarsInJobs(self):
 
     def testValidJobEnvs(self):
         """Validate jobs/config.json."""
-        with open(bootstrap.test_infra('jobs/config.json')) as fp:
+        with open(bootstrap.test_infra('jobs/config.json')) as fp, open(bootstrap.test_infra('jobs/validOwners.json')) as ownfp:
             config = json.loads(fp.read())
+            validOwners = json.loads(ownfp.read())
             for job in config:
+                # onwership assertions
+                self.assertTrue('sigOwners' in config[job], job)
+                self.assertTrue(type(config[job]['sigOwners']) == list)
+                self.assertTrue(len(config[job]['sigOwners']) > 0)
```

Comment:
Use implicit comparison:
https://google.github.io/styleguide/pyguide.html?showone=True/False_evaluations#True/False_evaluations

Also you can combine all this check with the In check:
`self.assertTrue(config[job].get('sigOwners') # Fail if sigOwners is missing or is []`